.yamllint.yml

```diff
@@ -0,0 +1,100 @@
+# Copyright 2023 The Authors (see AUTHORS file)
```

Comment:
I was thinking instead of using a entire .yamllint.yaml , we could use an optional workflow input variable like ignore path [here](https://github.com/abcxyz/pkg/blob/main/.github/workflows/yaml-lint.yml#L28).

Reply:
`ignore` can't be used as a flag, if we want make ignore as a flag in the cmd, the option is using `-d {ignore: /third_party/*}`. But `-d` and `-c` can't be used at the same time. See: https://github.com/abcxyz/lumberjack/actions/runs/5860727366/job/15889390357.

So I think having a `.yamllint.yml`, and `pkg` will use that yaml file to do lint by default should be the better option.

Reply:
We can also update`yamllint.yml` in abcxyz/pkg (https://github.com/abcxyz/pkg/blob/main/.yamllint.yml) to ignore any code in `third_party` directory if @sethvargo and @yolocs think this is a common pattern that can apply to more projects.

Reply:
Removed the `yamllint.yml`, as https://github.com/abcxyz/lumberjack/pull/441 solved the problem of lint from third_party packages.

Though I think it might worth adding `ignore` to `abcxyz/pkg/.yamllint.yml`

Reply:
Done